MRETestBed/Assets/Scenes/FunctionalTest-localhost.unity

```diff
@@ -851,8 +851,9 @@ MonoBehaviour:
     type: 2}
   MREURL: ws://localhost:3901
   testNames:
-  - asset-early-assignment-test
+  - asset-early-assignment
   - asset-preload-test
+  - mutable-asset-test
```

Comment:
>mutable-asset-test [](start = 4, length = 18)

Let's keep this list alphabetized.

Reply:
Can I rename the test then? I'd like to have all the asset tests together.

Reply:
Certainly!

Reply:
Okay, I've updated these to what they "should" be, but they'll be wrong until I change the names SDK-side.